packages.json

```diff
@@ -0,0 +1,393 @@
+[
+  {
+    "repo": "https://github.com/actions/setup-go",
+    "dependencies": [
+      "https://github.com/GlueOps/alertmanager",
+      "https://github.com/GlueOps/github-workflows"
+    ]
```

Comment:
to make sure I understand the data structure, the dependencies are the things that depend upon the repo, not the dependencies of the repo?

Reply:
yes. there is probably a better name for this. any thoughts?